src/atoms/Link/Link.web.js

```diff
@@ -50,7 +50,7 @@ const StyledText = styled(Text)`
   }
 `;
 
-const Link = ({ size, children, href, target, testID, rel, disabled }) => {
+const Link = ({ size, children, href, target, testID, rel, disabled, ...props }) => {
```

Comment:
Give some reasoning behind this?

Reply:
We are using custom components out of this with particular css at the consumer side.